Lib/exectrace/__init__.py

```diff
@@ -166,7 +170,7 @@ def run(self, entry_points=[0x0000]):
             self.schedule_entry_point(p, needs_label=True)
 
         self.restart_from_another_entry_point()
-        self.register_label(self.current_entry_point) # I think this is not needed.
```

Comment:
According to my tests, it is needed. Otherwise the entry point label is not registered.

Reply:
sounds good